geomstats/geometry/connection.py

```diff
@@ -41,39 +49,85 @@ def connection(self, tangent_vector_a, tangent_vector_b, base_point):
         raise NotImplementedError(
                 'connection is not implemented.')
 
-    def exp(self, tangent_vec, base_point):
+    def exp(self, tangent_vec, base_point, n_steps=N_STEPS):
```

Comment:
What is n_steps?

Reply:
the number of steps to discretize the time interval in the integration of the geodesic equation

Reply:
The number of steps to discretize the time interval in the integration scheme